rules.txt

```diff
@@ -208,6 +208,7 @@
 /artcommission(.*)                        301  /departments/philadelphia-art-commission/
 /art-commission                           301  /departments/philadelphia-art-commission/
 /artincityhall                            301  https://www.creativephl.org/exhibitions/
```

Comment:
add a catch-all here for the whole path /artincityhall(.*) and redirect it to the creativephl url